src/utils/sortAlphabetically.ts

```diff
@@ -0,0 +1,30 @@
+import { ProductSchema } from "@/schemas/ProductSchema";
+
+export const sortAlphabetically = (
+  items: ProductSchema[],
+  order: string = "ascending"
+): ProductSchema[] => {
+  return [...items].sort((a, b) => {
```

Comment:
Instead of making a copy of the array and using `sort`, use `toSorted`

```suggestion
  return items.toSorted((a, b) => {
```